editoast/src/views/timetable/round_trips.rs

```diff
@@ -0,0 +1,62 @@
+use crate::error::Result;
+use crate::views::pagination::PaginationQueryParams;
+use crate::views::pagination::PaginationStats;
+use crate::views::round_trips::RoundTrips;
+use crate::views::timetable::TimetableIdParam;
+use axum::Json;
+use axum::extract::Path;
+use axum::extract::Query;
+use serde::Serialize;
+use utoipa::ToSchema;
+
+crate::routes! {
+    "/round_trips" => {
+        "/train_schedules" =>  list_train_schedules,
+        "/paced_trains" => list_paced_trains,
+    },
+}
+
+#[derive(Serialize, ToSchema)]
+struct ListRoundTripsResponse {
+    #[serde(flatten)]
+    stats: PaginationStats,
+    results: RoundTrips,
+}
+
+/// Upsert a list of round trips / one-way of train schedules
+#[utoipa::path(
+    get, path = "",
+    tag = "timetable,round_trips",
+    params(TimetableIdParam, PaginationQueryParams<1000>),
+    responses((status = 200, description = "Paginated list of round trips / one-way of train schedules", body = inline(ListRoundTripsResponse)))
```

Comment:
nit: you can factorize the description in the documentation of the schema since you're inlining it both times